examples/forest-brawl/scenes/brawler.tscn

```diff
@@ -104,71 +104,71 @@ respawn_time = 8.0
 transform = Transform3D(0.5, 0, 0, 0, 0.5, 0, 0, 0, 0.5, 0, -0.5, 0)
 
 [node name="Skeleton3D" parent="bomber-guy/rig" index="0"]
-bones/0/position = Vector3(3.20746e-15, 0.583857, -0.0893679)
```

Comment:
I honestly don't think I did any change on this scene, but given I added some variables above, I kept the changes just in case. I opened the project with godot 4.1

If you wish, I can revert the changes on brawler.tscn